sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -304,7 +307,7 @@ def encode_to_stream(self, value, stream, nested):
       dict_value = value  # for typing
       stream.write_byte(DICT_TYPE)
       stream.write_var_int64(len(dict_value))
-      for k, v in dict_value.iteritems():
+      for k, v in dict_value.items():
```

Comment:
Any particular reason for the `iteritems()` -> `items()` change?

Reply:
`iteritems()` was replaced by `items()` in Python 3 dict

Reply:
`iteritems()` isn't available anymore in Python 3. Instead, `items()` returns an iterator. I've replaced it with `iteritems(dict)` from future.utils, which returns an iterator on both versions.